dartagnan/src/main/java/com/dat3m/dartagnan/encoding/EncodingContext.java

```diff
@@ -215,7 +216,7 @@ public Formula lastValue(MemoryObject base, int offset) {
             return formulaManager.getIntegerFormulaManager().makeVariable(name);
         }
         //TODO match this with the actual type.
```

Comment:
I guess we can now get rid of the comment

Reply:
I made a mistake and reverted my change. This comment is about encoding the content of a memory address (whose type we do not know) rather than encoding the address itself (whose type we know).